src/main/webapp/dev/js/main/instructorCourseEnrollHelper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Retrieves updated column header order and generates a header string.
+ *
+ * Example: Change this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section|Team|Name|Email|Comments"
+ *
+ * @param handsontableColHeader
+ * @returns {string} updated header string
+ */
+function getUpdatedHeaderString(handsontableColHeader) {
+    const colHeaders = handsontableColHeader.join('|');
+    return colHeaders.concat('\n');
+}
+
+/**
+ * Retrieves user data rows rows in the spreadsheet interface and transforms it into a string.
+ *
+ * Changes cell value null to ''. Filters empty rows in the process.
+ *
+ * Example:
+ * 2 by 5 spreadsheetData (before)
+ * ['TestSection1', 'Team1', 'null', 'test1@xample.com', 'test1comments']
+ * ['TestSection2', null, 'TestName2', 'test2@xample.com', null]
+ *
+ * 2 by 5 spreadsheetData (after)
+ * "TestSection1|Team1||test1@xample.com|test1comments\n
+ *  TestSection2||TestName2|test@example.com|\n"
+ *
+ * @param spreadsheetData
+ * @returns {string} user data rows
+ */
+function getUserDataRows(spreadsheetData) {
+    return spreadsheetData.map(row => (row.map(cell => (cell === null ? '' : cell))).join('|'))
+            .filter(row => row !== '||||') // remove empty rows
```

Comment:
Even though I suggested this, now I find it not that great because we're kind of hard-coding the number of columns in each row. I think a better way is to `filter` first before using `map`, and use `Array.prototype.reduce` to check that a row is not all `null`s